analytics.go

```diff
@@ -0,0 +1,72 @@
+package pagerduty
+
+import (
+	"bytes"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+)
+
+type AnalyticsRequest struct {
+	AnalyticsFilter *AnalyticsFilter `json:"filters,omitempty"`
+	AggregateUnit   string           `json:"aggregate_unit,omitempty"`
+	TimeZone        string           `json:"time_zone,omitempty"`
+}
+type AnalyticsResponse struct {
+	Data            []AnalyticsData  `json:"data,omitempty"`
+	AnalyticsFilter *AnalyticsFilter `json:"filters,omitempty"`
+	AggregateUnit   string           `json:"aggregate_unit,omitempty"`
+	TimeZone        string           `json:"time_zone,omitempty"`
+}
+
+type AnalyticsFilter struct {
+	CreatedAtStart string   `json:"created_at_start,omitempty"`
+	CreatedAtEnd   string   `json:"created_at_end,omitempty"`
+	Urgency        string   `json:"urgency,omitempty"`
+	Major          bool     `json:"major,omitempty"`
+	ServiceIds     []string `json:"service_ids,omitempty"`
```

Comment:
Please make this `ServiceIDs` per https://github.com/golang/go/wiki/CodeReviewComments#initialisms